libs/uv/uv_stubs.c

```diff
@@ -4,14 +4,15 @@
 #include <caml/fail.h>
 #include <caml/memory.h>
 #include <caml/mlvalues.h>
+// #include <caml/minor_gc.h>
 
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <uv.h>
 
 #if (UV_VERSION_HEX < (1 << 16 | 31 << 8))
```

Comment:
There is `UV_VERSION_MAJOR`, `UV_VERSION_MINOR`.

Reply:
That wouldn't make the check any simpler.